modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4872,22 +4921,41 @@ else if (Objects.equals(
 							String.valueOf(actualValues.get(entry.getKey())))));
 			}
 		}
+		else if (Objects.equals(
+					expectedEntry.getKey(),
+					"localizedMultiPicklistObjectFieldName")) {
```

Comment:
Curiosity: I did not find the condition that englobes the multiPicklistObjectFieldName. Is there a test?

Reply:
I know that is not the goal of this PR.

Reply:
Yes, there’s a test [here](https://github.com/pedro-oliveira446/liferay-portal/blob/73c469fc01680098a10f727ae4ece9da2203e976/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java#L3891-L3895). In this case, the value is passed as a ListEntry, which is the same type returned by the DTO, so there’s no need for a specific conditional to compare them.

Initially, I tried to follow this pattern, setting the localizedMultiPicklistObjectFieldName values as a ListEntry array. However, I encountered other challenges, and the test became significantly more complex. To simplify, I decided to keep the values as Arrays.asList(_listTypeEntryKey3, _listTypeEntryKey4).

Let me know if further clarification is needed.